packages/strapi/lib/configuration/hooks/dictionary/_plugins/index.js

```diff
@@ -129,10 +129,12 @@ module.exports = strapi => {
             controllers: plugin['controllers/*'],
             models: plugin['models/*'],
             services: plugin['services/*'],
-            policies: plugin['policies/*'],
             config: plugin['config/**']
           };
 
+          // Set policies value.
+          strapi.plugins[plugin.name].config.policies = plugin['config/policies/*'];
```

Comment:
@Aurelsicoko: not sure this is a good idea, by default it set an empty object, what is fine.

Exemple:
```js
{ controllers: { car: { index: [Function: index], identity: 'car', globalId: 'Car' } },
  services: { car: { identity: 'car', globalId: 'Car' } },
  config: { routes: [ [Object] ], policies: {} } }
```